# HG changeset patch
# User pm
# Date 1526647167 -7200
#      Fri May 18 14:39:27 2018 +0200
# Node ID 876ea25a4c2bf718a6649ebda99e3cf329ff8e4d
# Parent  9ac9e04a2d4f76897028646c4ca48dbcee05e26a
Ticket mantis 0004043: Le chemin du repertoire local xxx n'est pas approprie au stockage de messages

diff --git a/comm-esr60/comm/mailnews/base/prefs/content/AccountManager.js b/comm-esr60/comm/mailnews/base/prefs/content/AccountManager.js
--- a/comm-esr60/comm/mailnews/base/prefs/content/AccountManager.js
+++ b/comm-esr60/comm/mailnews/base/prefs/content/AccountManager.js
@@ -59,7 +59,7 @@
  */
 var gDangerousLocalStorageDirs = [
   // profile folder
-  { dirsvc: "ProfD",    OS: null },
+  { dirsvc: "ProfD",    OS: null, safeSubdirs:["Archives"] },
   // GRE install folder
   { dirsvc: "GreD",     OS: null },
   // Application install folder
